client/src/project/StateSelection.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <div class="row row-no-margin">
+    <div class="col-2fr">
+      <select class="full-width-block" v-model="selected" @blur="onBlur" @focus="onFocus" :class="{ error: hasError }">
+        <option v-for="state in states" :value="state.value">{{ state.text }}</option>
+      </select>
+    </div>
+    <div class="col-3fr">
+      <input v-show="selected === 'other'" class="full-width-block" type="text" placeholder="Please specify" v-model="other" @blur="onBlur" @focus="onFocus" :class="{ error: hasError }" />
```

Comment:
Using the same v-model won't work ?

Reply:
Nope. If they're the same v-model, then as soon as you change the "other" field it'll disappear since the value is no longer `'other'`.